README.md

```diff
@@ -461,10 +461,11 @@ Metrics from the subordinate process can help identify issues once the first met
 | sidecar.queue.running | gauge | number of running shards, those which have not exited | |
 | sidecar.queue.shards | gauge | number of current shards, as set by the queue manager | |
 | sidecar.queue.size | gauge | number of samples (i.e., points) standing in a queue waiting to export | |
-| sidecar.samples.processed | histogram | number of samples (i.e., points) read in a prometheus WAL batch | |
-| sidecar.samples.produced | counter | number of samples (i.e., points) read from the prometheus WAL | |
+| sidecar.samples.processed | histogram | number of metric samples read in a prometheus WAL batch | |
+| sidecar.samples.produced | counter | number of metric samples produced | |
 | sidecar.series.defined | counter | number of series defined in the WAL | |
 | sidecar.series.dropped | counter | number of series or metrics dropped | `key_reason`: various |
+| sidecar.points.produced | counter | number of points read from the prometheus WAL | |
```

Comment:
This PR gives me a desire to start over with these metriucs.

I think I no longer care to have a histogram at all, so I'd erase `sidecar.samples.processed`. Would you agree?

Then, the two remaining metrics have roles reversed.

We have other `sidecar.points.*` metrics that refer to OTLP point counts, so I think `sidecar.points.produced` should be the number of actual points produced. 

I am not sure we care to know how many samples Prometheus gave us, but I'd prefer to use `sidecar.samples.produced` to count those, since Prometheus uses "sample" to refer to these. The rate of `samples.produced` to `points.produced` tells us something about histogram size, but it's not clearly a high-value metric. What would you say to removing `sidecar.samples.produced`, leaving just `sidecar.points.produced` equaling the number of OTLP points output?

Reply:
sure, that works for me.